OnlineShopUI/online-shop-ui/src/components/Login.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import Modal from '@mui/material/Modal';
+import CloseIcon from '@mui/icons-material/Close';
+import './Login.scss';
+import FormGroup from '@mui/material/FormGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Checkbox from '@mui/material/Checkbox';
+import User from '../img/icon-user.png';
+
+export default function Login() {
+  const [open, setOpen] = React.useState(false);
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+
+  return (
+    <div>
+      <a href="#">
+        <img src={User} alt="user" className="log-btn" onClick={handleOpen} />
+      </a>
+      <Modal
+        className="log-modal"
+        open={open}
```

Comment:
Naming of variable 'open' is not self explained. If it's boolean type we better name it 'isOpen', and the best way is to describe all the logic variable provides - isLogInModalOpen